QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,91 @@
+
+
+This document is a declaration of software quality for the `libconsole-bridge-dev` ROS external dependency, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `libconsole-bridge-dev` Quality Declaration
+
+The ROS external dependency `libconsole-bridge-dev` **does not claim** to be in the **Quality Level 1** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [External dependencies requirements for Quality Level 1 in REP-2XXX](https://www.ros.org/reps/rep-2004.html).
+
+External dependencies are evaluated on a case by case basis, some items may not adhere to standards defined for ROS packages.
+
+## Version Policy
+
+### Version Scheme
+
+It is not publicly available the information regarding `libconsole-bridge-dev` uses a versioning system to manage automatically their version schemes. Their latest library release is set to 0.4.4
```

Comment:
Actually console_bridge_vendor uses console_bridge-0.4.1, not sure if a quality document should target the master of the library used or the explicit version used in the ros2 code.